frontend/src/App.js

```diff
@@ -1,55 +1,115 @@
-import logo from './logo.svg';
+import React, { Component } from "react";
+import axios from "axios"
 import './App.css';
 
-class Matchup {
-  constructor(team1, team2) {
-    this.team1 = team1
-    this.team2 = team2
```

Comment:
I thought about it a bit and this class was just completely unnecessary and complicating things